### PR TITLE
Minor errors in textfield

### PR DESCRIPTION
**Pull request by jperezgonzal**
## Description
Fixes en textfield y cambios mínimos en la testapp.

**Fixes en textfield**
- El helper y el counter se veía con el state "read-only"
- El padding izquierdo del textfield está mal con el state "read-only" (desalineado al label)
- En "disabled" y "read-only" el action no se pone en disabled.
- Error con el helper.

**Se cambio**
- Se agregó toast con el callback de "action"
- Se agregó la opción "None" para el "Prefix" y "Sufix".
- Cambió el nombre de "Prefix" y "Sufix" a "Left content" y "Right content" respectivamente.
- En la testapp, el counter se puso por defecto en 50.

## Checks
Are you sure you followed all those steps? In such case, let's say with me "I solemnly declare that ...":
- [ ] I have coded an example inside the Demo App,
- [ ] I have added proper tests,
- [x] I have modified the [Changelog](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CHANGELOG.md) file,
- [ ] I have updated GitHub wiki,
- [ ] I have the explicit approval of one or more members of the UX Team,
- [ ] I have updated the version in gradle.properties file.

## Change type
This is easy, let us know what this code is about:
- [ ] This code adds a new component
- [x] This code includes improvements to an existent component
- [ ] This code improves or modifies ONLY the Demo App.

## Check common errors
Whether you are an Android or iOS developer and if you're still there then we'll give you a present:
https://proandroiddev.com/how-to-maximize-androids-ui-reusability-5-common-mistakes-cb2571216a9f

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt [hunk 1]
```diff
@@ -43,7 +44,13 @@ internal object AndesSuffixTextfieldContent : AndesTextfieldContentInterface() {
 }
 
 internal object AndesPrefixTextfieldContent : AndesTextfieldContentInterface() {
-    override fun leftMargin(context: Context): Int = context.resources.getDimension(R.dimen.andes_textfield_prefix_left_margin).toInt()
+    override fun leftMargin(context: Context, state: AndesTextfieldState): Int {
+        return if (state == AndesTextfieldState.READONLY) {
```
Comment (lautimfernandez) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt:48]: podemos evitar este if en todos los casos haciendo un un state.leftMargin(context) y que el state devuelva lo que tenga qeu devolver

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/factory/AndesTextfieldConfigurationFactory.kt [hunk 2]
```diff
@@ -103,13 +104,14 @@ internal object AndesTextfieldConfigurationFactory {
...
     private fun resolveLeftComponent(context: Context, leftContent: AndesTextfieldContentInterface?): View? = leftContent?.component(context)
     private fun resolveRightComponent(context: Context, rightContent: AndesTextfieldContentInterface?): View? = rightContent?.component(context)
-    private fun resolveLeftComponentLeftMargin(context: Context, leftContent: AndesTextfieldContentInterface?): Int? = leftContent?.leftMargin(context)
+    private fun resolveLeftComponentLeftMargin(context: Context, state: AndesTextfieldState, leftContent: AndesTextfieldContentInterface?): Int? = leftContent?.leftMargin(context, state)
     private fun resolveLeftComponentRightMargin(context: Context, leftContent: AndesTextfieldContentInterface?): Int? = leftContent?.rightMargin(context)
-    private fun resolveRightComponentLeftMargin(context: Context, rightContent: AndesTextfieldContentInterface?): Int? = rightContent?.leftMargin(context)
+    private fun resolveRightComponentLeftMargin(context: Context, state: AndesTextfieldState, rightContent: AndesTextfieldContentInterface?): Int? = rightContent?.leftMargin(context, state)
```
Comment (lautimfernandez) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/factory/AndesTextfieldConfigurationFactory.kt:113]: aca deberiamos pasarla el stateInterface, para no tener que hacer state.state

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt [hunk 3]
```diff
@@ -134,7 +135,7 @@ internal object AndesActionTextfieldContent : AndesTextfieldContentInterface() {
 }
 
 internal object AndesIndeterminateTextfieldContent : AndesTextfieldContentInterface() {
-    override fun leftMargin(context: Context): Int = context.resources.getDimension(R.dimen.andes_textfield_indeterminate_left_margin).toInt()
+    override fun leftMargin(context: Context, state: AndesTextfieldState): Int = state.state.leftMargin(context)
```
Comment (lautimfernandez) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt:138]: cambiar solo por state.leftmargin , por parametro deberia recibir la interface

**Pull request merged**